### DISCUSSION TITLE
为什么只能下载480P

### DISCUSSION THREAD

**Discussion by 8lovearrowshoot** [leiurayer/downkyi#558] [Q&A]
我的downkyi只能最高下载480P清晰度了，试过很多个视频都是这样，需要做什么才能恢复

**Comment by keep1go** [comment]
为什么只能下载480P

**Comment by leiurayer** [comment]
B站修改了策略，登录就可以下载高画质了

**Reply by 14185638** [reply]
在downkyi登录你的B站账号后就可以下载更高清晰度了

**Reply by Stabil1ze** [reply]
但是下下来还是低画质的 比如4m30s 1080p的视频只有50M
